packages/terra-filter-view/docs/README.md

```diff
@@ -0,0 +1,32 @@
+# Terra Filter View
+
+A search component that displays user-provided results in a dropdown or result box.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-filter-view`
+
+## Usage
+
+```jsx
+import React from 'react';
+import FilterView from 'terra-filter-view';
+
+<FilterView>
```

Comment:
I feel we should drop the usage section. Now that we can view example code in the site, this section is not as valuable.